### ISSUE TITLE
Wizard does not preload debian default library paths

### ISSUE THREAD

**Issue by tehnick** [brezerk/q4wine#68]
Issue description:
http://bugs.debian.org/815018

Some useful links:
https://tracker.debian.org/pkg/wine
https://packages.debian.org/sid/amd64/libwine/filelist
https://packages.debian.org/sid/i386/libwine/filelist
https://packages.debian.org/sid/arm64/libwine/filelist
https://packages.debian.org/sid/armel/libwine/filelist
https://packages.debian.org/sid/armhf/libwine/filelist
https://packages.debian.org/sid/kfreebsd-i386/libwine/filelist
...
https://packages.debian.org/sid/all/wine/filelist
https://packages.debian.org/sid/all/wine-binfmt/filelist
https://packages.debian.org/sid/i386/wine32/filelist
https://packages.debian.org/sid/i386/wine32-preloader/filelist
https://packages.debian.org/sid/i386/wine32-tools/filelist
https://packages.debian.org/sid/amd64/wine64/filelist
https://packages.debian.org/sid/amd64/wine64-preloader/filelist
https://packages.debian.org/sid/amd64/wine64-tools/filelist

**Comment by tehnick** [comment]
Tiny note about Debian and Ubuntu multiarch scheme:
```
/usr/lib -> /usr/lib/<triplet>
/usr/lib/<pkgdir> -> /usr/lib/<triplet>/<pkgdir>
/usr/include: remains, used for arch-independent headers
/usr/include/<triplet>: used for arch-varying headers 
/usr/bin: no change
/usr/share: no change
/usr/sbin: no change
```

More details:
https://wiki.debian.org/Multiarch/Implementation#What_does_the_end_result_look_like.3F

**Comment by tehnick** [comment]
I have tried first strartup wizard in current version from master branch (1.2-r2-54-g5e03363) and have found few more problems. Let's see all process step by step:
![desktop_540](https://cloud.githubusercontent.com/assets/441190/13126239/c8947298-d5da-11e5-87da-a684a8279f69.png)
![desktop_541](https://cloud.githubusercontent.com/assets/441190/13126240/c8952602-d5da-11e5-9320-3868d66a5011.png)
![desktop_542](https://cloud.githubusercontent.com/assets/441190/13126247/c8f25d36-d5da-11e5-8b06-f8213aa89368.png)
![desktop_543](https://cloud.githubusercontent.com/assets/441190/13126242/c8b4e8c0-d5da-11e5-8aaa-5b67939a17eb.png)
![desktop_544](https://cloud.githubusercontent.com/assets/441190/13126241/c8b0b03e-d5da-11e5-8e1b-4329fcdb6fb3.png)
![desktop_545](https://cloud.githubusercontent.com/assets/441190/13126243/c8cd81dc-d5da-11e5-950a-3ec2e67e6cb6.png)
![desktop_546](https://cloud.githubusercontent.com/assets/441190/13126246/c8f08eac-d5da-11e5-8d9a-ed1d2fef5466.png)
![desktop_547](https://cloud.githubusercontent.com/assets/441190/13126244/c8cee216-d5da-11e5-974c-3c14ee5e4804.png)
![desktop_549](https://cloud.githubusercontent.com/assets/441190/13126245/c8e8336a-d5da-11e5-9f74-f24ec4211d73.png)

**Comment by tehnick** [comment]
Ok. Let's set the path to win64 libraries:
![desktop_550](https://cloud.githubusercontent.com/assets/441190/13126319/00c34252-d5db-11e5-9fca-8b809a80614e.png)
![desktop_551](https://cloud.githubusercontent.com/assets/441190/13126320/00dd1cfe-d5db-11e5-8202-6d876fb41fe9.png)
Nothing changed...

**Comment by tehnick** [comment]
BTW, first startup wizard dialog looks much better in stable release:
![desktop_552](https://cloud.githubusercontent.com/assets/441190/13126619/40a33c96-d5dc-11e5-9d37-f8663d34e923.png)
So issue #21 is not completely fixed yet.

**Comment by brezerk** [comment]
thx. will look into it :)

**Comment by brezerk** [comment]
hi @tehnick ,

Regarding to the links above:
```
/usr/lib/x86_64-linux-gnu/wine/libwine.so.1
/usr/lib/i386-linux-gnu/wine/libwine.so.1
```

can you show output of ```/sbin/ldconfig -p | grep libwine.so``` on your system please?
Also, it will be nice to see the output of q4wine compiled with -DDEBUG=ON option, if possible.

**Comment by tehnick** [comment]
> can you show output of /sbin/ldconfig -p | grep libwine.so on your system please?

Output is empty.

> Also, it will be nice to see the output of q4wine compiled with -DDEBUG=ON option, if possible.

Probably later. That do you want to find there?

**Comment by brezerk** [comment]
> Probably later. That do you want to find there?
> Output is empty.

I guess this is the root cause :)
Does ```/sbin/ldconfig -p | grep -i wine``` returns anything?

**Comment by tehnick** [comment]
```
$ /sbin/ldconfig -p | grep wine
        libkwineffects.so.6 (libc6) => /usr/lib/i386-linux-gnu/libkwineffects.so.6
$ /sbin/ldconfig -p | grep wine
        libkwineffects.so.6 (libc6,x86-64) => /usr/lib/x86_64-linux-gnu/libkwineffects.so.6
```

**Comment by brezerk** [comment]
*sigh* just wonder why there is no libwine.so =\

**Comment by tehnick** [comment]
> just wonder why there is no libwine.so =\

Probably because this is wine-specific library. See: https://github.com/brezerk/q4wine/issues/68#issuecomment-185396471
```
...
/usr/lib/<pkgdir> -> /usr/lib/<triplet>/<pkgdir>
...
```

**Comment by brezerk** [comment]
@tehnick does your system have any wine-devel pkg or so? If yes, can you install it and see if ```/sbin/ldconfig -p | grep libwine.so``` will work?

**Comment by brezerk** [comment]
well. on the other hand, it seems like there is no need to strictly require this option:

```
       WINEDLLPATH
              Specifies the path(s) in which to search for builtin dlls and Winelib applications. This is a list of directories separated by
              ":". In addition to any directory specified in WINEDLLPATH, Wine will also look in /usr/lib32/wine.
```

So option allows *additionally* specify extra paths where to look at. It will use default path anyway.

**Comment by brezerk** [comment]
Should be fixed with: https://github.com/brezerk/q4wine/commit/d3b5c0f1e587e621a1a659e2dee4f1c2306c4472

**Comment by tehnick** [comment]
> does your system have any wine-devel pkg or so?

Yes. libwine-dev

> If yes, can you install it and see if /sbin/ldconfig -p | grep libwine.so will work?

No.

> Should be fixed with: d3b5c0f

Ok. It works for me. Thanks.
